### DISCUSSION TITLE
Tuning hyperparameter with Akaike information Criterion

### DISCUSSION THREAD

**Discussion by melitong** [keras-team/keras-tuner#533] [General]
Hello, i am new in keras-tuner. I have tried hyperparameter tuning with keras-tuner to maximize the accuracy of validation set. 
However, I have read about Akaike Information Criterion (AIC), it can be used to select best model. Can i use the AIC as a metrics or objective function in hyperparameter tuning and get best model with minumum AIC in keras-tuner?

**Comment by haifeng-jin** [comment]
Yes, you may need to use `objective=kerastuner.Objective('val_aic', direction='max'),` and define a custom metric named `'aic'` and pass to the tuner.
We will add this tutorial to the docs.
